test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -24,13 +24,17 @@
               :localizations {:en {:title "en title"
                                    :infourl "hello"}
                               :fi {:title "fi title"
+                                   :infourl nil}
+                              :sv {:title "sv title"
```

Comment:
You shouldn't need to sv here, `languages` is `[:en :fi]`. This makes the cljs tests fail on circleci.

Reply:
ok, well, at least if I do not include those, they fail locally 😆

Reply:
that's super weird since `build-request` is a pure function and this test fixes all inputs